### ISSUE TITLE
If AMO credentials fail, offer unsigned

### ISSUE THREAD

**Issue by Madis0** [Noitidart/Chrome-Store-Foxified#83]
If the extension is unable to connect to AMO in order to sign the extension, it should still offer the user to install the unsigned version (+ tutorial on how to do so). 

I've noticed the extension to still appear on the dashboard, but you can't expect other users to notice the same...

**Comment by Noitidart** [comment]
It should currently already offer "Install Unsigned":

![](https://screenshots.firefoxusercontent.com/images/556ce5cd-c362-44fb-9387-81338a6fedcb.png)

Is this not showing up?

I need to definitely add a popup/tutorial though.

**Comment by Madis0** [comment]
No, the dashboard has both "install unsigned" and a tutorial. I was talking about this pointless page:

![](http://i.pi.gy/nJK4k.png)

**Comment by Noitidart** [comment]
Oh!!! That's real bad it should never happen. What this means is that it was not able to find the "user key" and "user secret" from this page: https://addons.mozilla.org/en-US/developers/addon/api/key/

Are you able to reproduce this? I haven't been able to get this error to show up.

**Comment by Madis0** [comment]
Yes, I have got that for every extension. 
The reason is probably that I browse Chrome Web Store in a defined Container but AMO in a generic Container, haven't bothered to change that 😁

**Comment by Noitidart** [comment]
Oh way interesting I'll try that out, for sure if AMO is in a generic container, it is not able to get those cookies, but it should give error "not logged in". I'll try to reproduce that, thanks very much for your fast reply!

**Comment by Noitidart** [comment]
This issue is not the same as but related to - https://github.com/Noitidart/Chrome-Store-Foxified/issues/89

For #89 I have a fix coming up for 3.3 - but in case it still fails (it should never fail) we can create a dialog. But because it should never ever fail we can put this to low priority.

However it would be maybe nice to stick in somewhere why it needs to connect to AMO. Definitely open to ideas on where to put that, and definitely open to contributions adding this in! :)
